Sources/Player/Common/DRM/AVContentKeyRequest+KeyRequest.swift

```diff
@@ -21,6 +21,7 @@ extension AVContentKeyRequest: KeyRequest {
 		try await withCheckedThrowingContinuation { continuation in
 			let completionHandler: (Data?, Error?) -> Void = { spc, error in
 				if let spc {
+					PlayerWorld.logger?.log(loggable: PlayerLoggable.avcontentKeyRequestNoData)
```

Comment:
Is this correct? the condition does not seem to be an error, rather, the normal flow.

Reply:
hmm, let me check, I wanted to check in the bad scenarios, maybe added in the wrong place, I'll review it.

Reply:
ok, I think I meant to add in the else, but we already have error there, and we don't have any issue with this specific error I believe so I'll delete it.

Reply:
remove ✅